site/_posts/2020-03-31-Announcing-Jansa-Alpha.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Announcing Jansa Alpha Release
+author: chessbyte
+date: 2020-03-31 19:12:13 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+We have just built Jansa Alpha. This release has 2 major changes that are included in the [roadmap](https://github.com/orgs/ManageIQ/projects/13#column-7337345)
+
+- Podification
+- VMware VimBroker Removal - described [here](http://talk.manageiq.org/t/vmware-vimbroker-removal/4775)
```

Comment:
- VMware VimBroker Removal - In addition to the various performance issues with the VMware VimBroker, one of the underlying technologies, [DRb](https://en.wikipedia.org/wiki/Distributed_Ruby) was incompatible with Podification.  As such, we redesigned the VMware interaction in such a way that the broker was removed completely.  Much more detail can be found [here](http://talk.manageiq.org/t/vmware-vimbroker-removal/4775).